### ISSUE TITLE
node api usage documentation

### ISSUE THREAD

**Issue by eankeen** [htmlhint/HTMLHint#450]
**Is your feature request related to a problem? Please describe.**
A clear and concise description of what the problem is.
haii, i sort of perused the documentation to find examples of how to use this tool using the node api, but couldn't find it - is this something that can be added to the documentation website, or does it exist somewhere that i couldn't find

**Describe the solution you'd like**
like maybe a simple explanation of the most commonly used exported functions

**Describe alternatives you've considered**
n/a

thankies,
edwin

**Comment by Shinigami92** [comment]
On the one hand we have this new website: https://htmlhint.com/docs/user-guide/getting-started
It's currently under construction

But if you want to now what to `import { /**/ } from 'htmlhint'`
we are currently working on rebuilding everything with TypeScript 🙂

Currently you should use something like

```js
const HTMLHint = require('htmlhint').HTMLHint
HTMLHint.verify(html, options)
```

**Comment by eankeen** [comment]
ohh, thanks! totally missed that :P

**Comment by 3cp** [comment]
The changed api has broken lots of htmlhint integration. The codemirror standard plugin doesn't work, so is vscode https://github.com/microsoft/vscode-htmlhint/issues/76

**Comment by 3cp** [comment]
Looks like it's not the api change, it's the change on esm default export.

**Comment by Shinigami92** [comment]
> Looks like it's not the api change, it's the change on esm default export.

Sorry that we have not replied to you in the last 5 days

Yes, we made some changes to the export strategy and yes, this may have damaged some plugins

But that is fine for us in the current state, because
1. htmlhint is in a `0.x.x` / `alpha` state
2. We have released versions `0.13` and `0.14`, in which version `0.x.*` may contain such changes

We are currently aiming for version `1.0.0` and with this first non-alpha/beta state many/all plugins may need to be updated, especially because we will have a new configuration structure and will no longer support the current one.
